src/pages/index.astro

```diff
@@ -28,8 +28,8 @@ const links = [
   <Hero client:load />
   <ProductsCodePush client:load />
   <Divider />
-  <Team client:load />
   <Pricing client:load />
+  <Team client:load />
```

Comment:
moved Team to after Pricing as discussed.

Reply:
Yeah, the purpose of team is validation.  It's just "oh and btw, you can trust these folks"